### PR TITLE
adding atom list screen

### PR DESCRIPTION
**Pull request by lomudi**
adding a screen the present a simple list of an atom feed - it can be shown by adding the UI tag 'generic_atomlist' to the relevant category in Applicaster2.
simulator screen shot 1 may 2016 4 20 05 pm
simulator screen shot 1 may 2016 4 20 34 pm

### CODE REVIEW

File: AtomListScreen_iPhone.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   "style":{
+      "layout_name":"collection_standard",
+      "cs_tag":null
+   },
+   "attributes":{
+      "line_spacing":5,
+      "interitem_spacing":6,
+      "inset_top":5,
+      "padding_left": "${${ZAPP_STYLES}.appDefine.isRTL} ? 0 : 0",
+      "padding_right": "${${ZAPP_STYLES}.appDefine.isRTL} ? 0 : 0"
```
Comment (RoiKed) [inline AtomListScreen_iPhone.json]: remove

File: AtomListScreen_iPhone.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+         "type":"cell_types",
+         "style":{
+            "layout_name":"cell_fuse_list",
+            "cs_tag":null
+         },
+         "attributes": {
+             "height_pixel": 87,
+             "width_pixel": 320,
+             "padding_left": 5,
+             "padding_right": 5,
```
Comment (RoiKed) [inline AtomListScreen_iPhone.json]: remove padding, make width 310 and add inset_right as 5 in the collection

File: AtomListScreen_iPad.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      {
+         "type":"cell_types",
+         "style":{
+            "layout_name":"cell_fuse_list",
+            "cs_tag":null
+         },
+         "attributes":{
+            "height_pixel":105,
+            "width_pixel":330,
+            "image_tag":"image_1",
```
Comment (RoiKed) [inline AtomListScreen_iPad.json]: change to image_base

**Comment by lomudi** [comment]
@RoiKed done

**Comment by RoiKed** [comment]
💯

**Pull request merged**